### ISSUE TITLE
Using ..\ does not move to previous folder in Characters section of select.def

### ISSUE THREAD

**Issue by jenngra505** [ikemen-engine/Ikemen-GO#2198]
### Describe the bug

I have been trying to consolidate my Mugen and IKEMEN rosters by sharing them in a separate main roster folder, but IKEMEN only accepts the format of using ..\ for the previous folder on the ExtraStages section.

### To Reproduce

1. Add a character to a subdirectory just outside your IKEMEN folder
2. Use `..\..\Example\chars\kfm\kfm.def`
3. Have a blank slot appear where there shouldn't be one.

### Expected behavior

Character will load from the chosen subdirectory.

### Engine Version (or source code date)

0.99, 2024-12-19

### Operating system

Linux

**Comment by potsmugen** [comment]
Does Mugen accept that? I just write the whole path.

**Comment by jenngra505** [comment]
Yes, Mugen accepts that but only for characters. It's why I reported this bug in the first place.

**Comment by K4thos** [comment]
can't reproduce. Tested with kfm outside ikemen folder and adding ..\kfm\kfm.def to system.def
